ocs_ci/helpers/managed_services.py

```diff
@@ -31,6 +31,15 @@
 log = logging.getLogger(__name__)
 
 
+class ResourceWithPhase(OCP):
```

Comment:
_has_phase is declared as private variable in the class OCP. But many places outside the class alter it using "_has_phase=True". Eventhough this class helps to use the variable in an appropriate way, as this is not specific for managed services please move it to a different module.